test/test-services.env

```diff
@@ -1,19 +1,32 @@
 # Any of these can be overridden locally in ../local.env
```

Comment:
No values are changed below, but reorganised to bring the URL and HOST values near to each other and to the top. Some additional commenting.